.circleci/config.yml

```diff
@@ -1663,6 +1667,38 @@ jobs:
           binary_path: "build/solc/solc"
           preset: "<< parameters.preset >>"
 
+  b_bytecode_osx_x86_64:
+    parameters:
+      preset:
+        type: string
+    <<: *base_osx
+    parallelism: 2 # For prepare_bytecode_report
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - prepare_bytecode_report:
+          label: "osx_x86_64"
+          binary_type: x86-64
+          binary_path: "build/solc/solc"
+          preset: "<< parameters.preset >>"
+
+  b_bytecode_osx_x86_64h:
```

Comment:
Again, I'd remove the haswell job altogether.

Reply:
Why? I think it cannot hurt to have also these jobs.

Reply:
I removed that: we dont have any special `x86_64h` code inside our universal binary.